### PR TITLE
Bundle.js mapped to a specific port, it should work on all given ports

### PR DESCRIPTION
**Pull request by ooade**
#### Pre-Submission Checklist

- [x] Your pull request targets the `staging` branch of FreeCodeCamp.
- [x] Branch starts with either `fix/`, `feature/`, or `translate/` (e.g. `fix/signin-issue`)
- [x] You have only one commit (if not, [squash](http://forum.freecodecamp.com/t/how-to-squash-multiple-commits-into-one-with-git/13231) them into one commit).
- [x] All new and existing tests pass the command `npm run test-challenges`. Use `git commit --amend` to amend any fixes.
#### Type of Change

- [ ] Small bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds new functionality)
- [ ] Breaking change (fix or feature that would change existing functionality)
- [ ] Add new translation (feature adding new translations)
#### Checklist:

- [x] Tested changes locally.
- [x] Closes currently open issue (replace XXXX with an issue no): Closes #XXXX

### CODE REVIEW

**Comment by marhyorh** [comment]
I think this only applies to staging branch :smile:

File: webpack.config.js [hunk 1]
```diff
@@ -16,7 +16,7 @@ module.exports = {
       'bundle-[name].js' :
       'bundle-[name]-[chunkhash].js',
     path: path.join(__dirname, '/public/js'),
-    publicPath: __DEV__ ? 'http://localhost:2999/js' : '/js'
+    publicPath: __DEV__ ? `http://localhost:${process.env.PORT || 3000}/js` : '/js'
```
Comment (BerkeleyTrue) [inline webpack.config.js]: This can't be the same port as the server

Comment (marhyorh) [inline webpack.config.js]: And why is that??

Comment (BerkeleyTrue) [inline webpack.config.js]: Because it is served by the webpack dev server not the main server. That is why it is on a different port.

**Comment by BerkeleyTrue** [comment]
What is the issue you ran into that you needed this change?

**Comment by marhyorh** [comment]
I don't see the codemirror editor, the map sidebar don't show so i checked the console and saw it was requiring for `http://localhost:2999/js/bundle.js`, which does not exist

**Comment by BerkeleyTrue** [comment]
Looks like you'll need to investigate why the webpack dev server isn't booting up. This PR won't fix that.

Try blowing away your node_modules and starting `gulp` again.

Send me a message on Gitter if you are still hitting errors.


Happy Coding

**Comment by marhyorh** [comment]
Okay :smile:

**Pull request closed without merging**